modules/cluster-logging-forwarding-json-logs-to-the-default-elasticsearch.adoc

```diff
@@ -1,7 +1,14 @@
 [id="cluster-logging-forwarding-json-logs-to-the-default-elasticsearch_{context}"]
 = Forwarding JSON logs to the Elasticsearch log store
 
-For the Elasticsearch log store that OpenShift Logging manages, you must create a different index for each format in advance if your JSON log entries _have different formats_. Otherwise, forwarding different formats to the same index can cause type conflicts and cardinality problems.
+For an Elasticsearch log store, if your JSON log entries _follow different schemas_, configure the `ClusterLogForwarder` custom resource (CR) to group each JSON schema into a single output definition. This way, Elasticsearch uses a separate index for each schema.
```

Comment:
I don't think the emphasis on "follow different schemas" is necessary. But I see it was similarly done in the previous text, so I won't hold up the PR over it or anything.

Reply:
We don't want users to fuss over this procedure if their JSON log entries all follow one schema. I'd welcome suggestions on a better way to word this.